packages/dcos-integration-test/extra/conftest.py

```diff
@@ -0,0 +1,482 @@
+import collections
+import logging
+import os
+import uuid
+
+import dns.exception
+import dns.resolver
+import pytest
+import requests
+import retrying
+
+LOG_LEVEL = logging.INFO
+TEST_APP_NAME_FMT = '/integration-test-{}'
+# If auth is enabled, by default, tests use hard-coded OAuth token
+AUTH_ENABLED = os.getenv('DCOS_AUTH_ENABLED', 'true') == 'true'
+# Set these to run test against a custom configured user instead
+LOGIN_UNAME = os.getenv('DCOS_LOGIN_UNAME')
+LOGIN_PW = os.getenv('DCOS_LOGIN_PW')
+
+
+@pytest.fixture(scope='module')
+def cluster():
```

Comment:
In our closed source we refer to this as `dcos`` and here as `cluster`. We should homogenize this naming convention.